### PR TITLE
Remove the console logpoint icon

### LINKED ISSUE #3521: Remove the gray-breakpoint logpoint indicator

### PR DESCRIPTION
**Pull request by jaril**
Fix #3521.

### CODE REVIEW

File: src/devtools/client/webconsole/components/Output/MessageIcon.js [hunk 1]
```diff
@@ -27,12 +27,9 @@ function getIconElement(level, onRewindClick, type) {
   let title = l10n.getStr(l10nLevels[level] || level);
   const classnames = ["icon"];
 
-  if (type === "logPoint") {
-    classnames.push("logpoint");
```
Comment (ryanjduffy) [inline src/devtools/client/webconsole/components/Output/MessageIcon.js:31]: Is there a CSS class that needs removed as well?

File: src/devtools/client/webconsole/components/Output/MessageIcon.js [hunk 2]
```diff
@@ -27,12 +27,9 @@ function getIconElement(level, onRewindClick, type) {
...
-  } else if (type === "paywall") {
+if (type === "paywall") {
```
Comment (ryanjduffy) [inline src/devtools/client/webconsole/components/Output/MessageIcon.js:30]: Nit: missing leading spaces

**Review by ryanjduffy — APPROVED** [review]

File: src/devtools/client/webconsole/components/Output/MessageIcon.js (hunk 1 above)
Comment (jaril) [reply src/devtools/client/webconsole/components/Output/MessageIcon.js:31]: yep! fixed

File: src/devtools/client/webconsole/components/Output/MessageIcon.js (hunk 2 above)
Comment (jaril) [reply src/devtools/client/webconsole/components/Output/MessageIcon.js]: fixed!

**Comment by ryanjduffy** [comment]
:shipit:

**Pull request merged**